src/http/sr-modules-keys/sr-modules-keys.service.ts

```diff
@@ -40,7 +41,7 @@ export class SRModulesKeysService {
 
   async getModuleKeys(
     moduleId: string | number,
-    filters: KeyQuery,
+    filters: KeysFilter,
```

Comment:
lets use types from common/entities better

Reply:
OK, I agree. I've just updated this type.